README.md

```diff
@@ -48,13 +69,13 @@ This app has an interface that describes which rules must be implemented by a bl
   }
 }
 ```
-The minicart has as a required block the `product-summary`. So, any minicart implementation created must add a product-summary as a block that is inside of minicart. To know how is the API of `product-summary` see the next section.
+The minicart has as a required block the `product-summary`. So, any minicart block implementation created must add a product-summary as a block that is inside of minicart. To know how is the API of `product-summary` see the next section.
 
 #### Configuration 
 Through the Storefront, you can change the minicart's behavior and interface. However, you also can make in your theme app, as Dreamstore does.
 
-| Prop name          | Type       | Description                                                                 |
-| ------------------ | ---------- | --------------------------------------------------------------------------- |
+| Prop name                   | Type       | Description                                                        |
```

Comment:
I think it is important to add a `Default value` field in the configuration table.
e.g.

```
| Prop name | Type | Description  | Default value
| ------------------ | ---------- | ---------------------- |
| `type` | `String`| Define Minicart mode. (values: 'popup' or 'sidebar')| `popup`
```